lang/en/auth_saml2.php

```diff
@@ -110,6 +110,7 @@
 $string['multiidp:label:defaultidp'] = 'Default IdP';
 $string['multiidp:label:admin'] = 'Admin';
 $string['multiidp:label:whitelist'] = 'Redirected IP addresses';
+$string['multiidp:label:whitelist_help'] = 'If set, it will force clients to this IdP. Format: xxx.xxx.xxx.xxx/bitmask. Separate multiple subnets with \',\' (comma).';
```

Comment:
the delimiter is just \n ?

https://github.com/moodle/moodle/blob/master/lib/classes/ip_utils.php#L236

Reply:
Ah. I Was using address_in_subnet(), not \core\ip_utils::is_ip_in_subnet_list, I'll adjust that.